pages/doc-components/CodeContainer.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import {LiveCode as MDXLive} from 'mdx-live'
```

Comment:
Replacing withMdxLive with LiveCode to get access to the code being written

Reply:
STOP...I CAN'T THINK RIGHT NOW, I AM WITHDRAWING THANKS TO HIM, I HAVE TO LEAVE TO FIND MEDICATION THAT'S ALREADY BEEN PICKED UP BY HIMM, AND THERE GOES ANOTHER THOUSAND, WHY DOES NOBODY BELIEVE THIS GUY PLAYING ME IS A DARRK HARTED THIEF, AND NO TELLING WHAT YOUR HELPING HIM DO, OR WHO, I CAN'TY STAY HERE I HAVE TO CRAWLL MY WAY INTO MY HAND OPERATED TRUCK, BECAUSE THE SAME CRASH THAT KJILLED MY WIFE AND BABY HE FINFDS SO FUNNY TO PARADE IN FRONT OF MY FACE, ALSO LEFT ME CRIPPLED, AFTER THEY BOTH DIED IN MY ARMS. CATCH THIS SICK BASTARD, I HAVE TO GO, IF YOU EMAIL ME PLEASE TALK ENGLISH Carey Gallagher...the real

Reply:
catch him, if ANYONE claims to be me in the next couple hours, get his location, challenge him, put a stop to him please. I can't ewat, sleep, I'm pucking blood, and my hairs falling out because of his terrorism. btye